src/Modules/SpectatorCamModeModule/Config/ISpectatorCamModeSettings.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel;
+using Config.Net;
+using EvoSC.Common.Util.Manialinks;
+using EvoSC.Modules.Attributes;
+
+namespace EvoSC.Modules.Official.SpectatorCamModeModule.Config;
+
+[Settings]
+public interface ISpectatorCamModeSettings
```

Comment:
I think adding a config option to enable/disable the custom spectator switch would be useful, or move that functionality to a separate module.

Reply:
wdym it is a independent module, that can be disabled?